### PR TITLE
Swap two nibbles in C

### PR DESCRIPTION
**Pull request by ag278**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This is code to swap nibbles in C using bitwise operator as a part of CH20

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#980

### CODE REVIEW

**Comment by ag278** [comment]
@iamrajiv @siddharth25pandey @Kajol-Kumari please review my PR

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ag278 Name the program like this `Swap_two_nibbles.c`.

File: C/other/Swap two nibbles.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    int result = swapnibbles(num);
+    printf(" %d\n",result);
+}
+
+/*
+Input:
+Enter a number: 50
+Output: 
+35
+*/
```
Comment (Kajol-Kumari) [inline C/other/Swap two nibbles.c]: please provide one blank line at the end of the file.

File: C/other/Swap two nibbles.c [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+int main()
+{   
+    //taking input
+    int num;
+    printf(" Enter a number : \n");
+    scanf("%d",&num);
+    int result = swapnibbles(num);
+    printf(" %d\n",result);
```
Comment (Kajol-Kumari) [inline C/other/Swap two nibbles.c]: Please provide a message like `output` before printing the result here.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@ag278 please incorporate the following changes.

**Comment by ag278** [comment]
@iamrajiv @Kajol-Kumari I have solved the issues. Please verify it

**Review by iamrajiv — APPROVED** [review]

File: C/other/Swap_two_nibbles.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    printf(" Enter a number : \n");
+    scanf("%d",&num);
+    int result = swapnibbles(num);
+    printf("output is  %d\n",result);
+}
+
+/*
+Input:
+Enter a number: 50
+Output:
```
Comment (Kajol-Kumari) [inline C/other/Swap_two_nibbles.c:23]: please change this line to `output is: 35` as you have made the change in the code here.

**Review by Kajol-Kumari — APPROVED** [review]
@ag278 Minor change required, Rest LGTM :+1:

**Review by siddharth25pandey — APPROVED** [review]
@ag278 Good to go
Able to merge.
Thanks!

**Comment by siddharth25pandey** [comment]
@all-contributors please add @ag278 for his code

**Pull request merged**